mysql-test/t/disabled.def

```diff
@@ -17,3 +17,4 @@ log_tables-big           : Bug#11756699 2010-11-15 mattiasj report already exist
 ds_mrr-big @solaris      : Bug#14168107 2012-04-03 Hemant disabled new test added by Olav Sandstå,since this leads to timeout on Solaris on slow sparc servers
 mysql_embedded_client_test	: Bug#13964673 2012-04-16 amitbha since most of the test cases are failing
 mysql_client_test_embedded : Bug#16084066 2013-01-08 Disabled since this test is failing
+main.unique_thread_id : Temporary
```

Comment:
this test is not useful. Manuel had added thread id collision logic, which had increased contention but was never proven to be the original issue (from him). 
If thread id collision is an issue, we can increase thread id to 64 bits and that way the wraparound will take years.